core/src/test/java/org/elasticsearch/common/network/NetworkModuleTests.java

```diff
@@ -41,11 +33,12 @@
 import org.elasticsearch.rest.RestChannel;
 import org.elasticsearch.rest.RestRequest;
 import org.elasticsearch.rest.action.cat.AbstractCatAction;
-import org.elasticsearch.tasks.Task;
 import org.elasticsearch.test.transport.AssertingLocalTransport;
 import org.elasticsearch.transport.Transport;
 import org.elasticsearch.transport.TransportService;
 
+import java.util.Collections;
```

Comment:
No code was added to this test class, only removed, so is this added import necessary?